snapshots/output/syntax/src/inheritance.ts

```diff
@@ -12,7 +12,7 @@
 //  documentation ```ts\n(property) property: string\n```
     interfaceMethod(): string
 //  ^^^^^^^^^^^^^^^ definition syntax 1.0.0 src/`inheritance.ts`/Superinterface#interfaceMethod().
-//  documentation ```ts\n(method) interfaceMethod() => string\n```
+//  documentation ```ts\n(property) interfaceMethod: () => string\n```
```

Comment:
I have no clue why this changed. I don't think it's a big problem but it is concerning that I can't explain why it changed 🤔

Reply:
Fixed now.